config/settings.py

```
@@ -13,11 +13,13 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
 
-SECRET_KEY = os.environ.get("DJANGO_SECRET_KEY", None)
+SECRET_KEY = getenv(
+    "DJANGO_SECRET_KEY", "insecure_secret_key"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)